src/utils/explorers.ts

```diff
@@ -152,7 +152,7 @@ export async function queryExplorerForTx(
 
 export async function queryExplorerForTxReceipt(
   multiProvider: MultiProvider,
-  chainId: ChainId,
+  domainId: DomainId,
```

Comment:
chainName would probably be better if we're going through the trouble of changing these but def ndb

Reply:
i tried to minimise diff but yeah happy to go with chainName - it's my preferred key too